src/client/Microsoft.Identity.Client/Internal/Requests/SilentRequest.cs

```diff
@@ -191,7 +191,7 @@ private async Task<MsalTokenResponse> TryGetTokenUsingFociAsync(CancellationToke
                 {
                     // Hack: STS does not yet send back the suberror on these platforms because they are not in an allowed list,
                     // so the best thing we can do is to consider all errors as client_mismatch.
-#if NETSTANDARD || UAP || MAC
+#if NETSTANDARD || WINDOWS_APP || MAC
                     ex?.GetType();  // avoid the "variable 'ex' is declared but never used" in this code path.
```

Comment:
>ex?.GetType(); [](start = 20, length = 14)

should probably have been fixed by not specifying the variable in the catch paragraph instead of wasted cpu time?